workers/unity/Assets/Fps/Scripts/GameLogic/WorldTiles/TileFootprintHelper.cs

```diff
@@ -0,0 +1,89 @@
+﻿using UnityEngine;
+#if UNITY_EDITOR
+using UnityEditor;
+using UnityEditor.Experimental.SceneManagement;
+
+#endif
+
+
+public class TileFootprintHelper : MonoBehaviour
```

Comment:
what exactly does this MB do?

Reply:
I used it when authoring geometry within tiles to ensure I kept within a 'safe space' from the origin. I'll add a comment.